kv/migration/all/all.go

```diff
@@ -31,5 +31,9 @@ var Migrations = [...]migration.Spec{
 	Migration0011_PopulateDashboardsOwnerId,
 	// Populate the DBRP service ByOrg index
 	Migration0012_DBRPByOrgIndex,
+	// repair DBRP owner and bucket IDs
+	Migration0013_RepairDBRPOwnerAndBucketIDs,
+	// reindex DBRPs
+	Migration0014_ReindexDBRPs,
```

Comment:
The reindex here is required for the `Find` methods in the DBRP service to locate repaired DBRPs. Reordering the migrations so my new `Migration0013` runs before the existing `Migration0012` also works, but that felt like a potentially-dangerous idea.